DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/src/CMakeLists.txt

```diff
@@ -32,15 +32,15 @@ if(FPGA_BOARD MATCHES ".*a10.*")
     set(COMPLEX 1)
     set(FIXED_ITERATIONS 64)
     set(CLOCK_TARGET 360MHz)
-    set(SEED "-Xsseed=5")
+    set(SEED "-Xsseed=26")
```

Comment:
Assuming you started at 1, that's a lot of seeds ;)

Reply:
Unfortunately it is; rough history of seeds for A10: 10->13->5->13->31->7->10
Getting the 24k matrices/s threshold requires a very good seed with this source code.